<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>522</width>
    <height>895</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QGroupBox" name="groupBox_6">
      <property name="title">
       <string>Session:</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLineEdit" name="workDirectoryLE">
           <property name="placeholderText">
            <string>Work directory</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="workDirSelectB">
           <property name="text">
            <string>Select</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLineEdit" name="commandLE">
         <property name="text">
          <string>hashcat</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Session name:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="sessionNameLE"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Attack mode:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="attackModeComboBox"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Speed:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="speedComboBox"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="restoreSessionCB">
         <property name="text">
          <string>Restore session</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="bruteforceGroupBox">
      <property name="title">
       <string>Bruteforce:</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_8" stretch="0,0,0,0,0">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Charset pattern:</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLineEdit" name="charsetPatternLE"/>
         </item>
         <item>
          <widget class="QPushButton" name="charsetPatternAddB">
           <property name="text">
            <string>Add</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Charset list:</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QListWidget" name="charsetsListW"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="1,0">
         <item>
          <widget class="QLineEdit" name="bruteforceMaskLE">
           <property name="placeholderText">
            <string>?1?l?u?d?h?H?s?a?b</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="helpB">
           <property name="text">
            <string>?</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="hashFileGropBox">
      <property name="title">
       <string>Hash:</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_11">
         <item>
          <widget class="QLineEdit" name="hashFileLE">
           <property name="placeholderText">
            <string>hashFile</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="hashFileSelectB">
           <property name="text">
            <string>Select</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_12" stretch="1,0">
         <item>
          <widget class="QComboBox" name="hashTypeComboBox"/>
         </item>
         <item>
          <widget class="QSpinBox" name="hashTypeSpinBox">
           <property name="maximum">
            <number>9999999</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="rulesGroupBox">
      <property name="title">
       <string>Rules file:</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="ruleFileLE"/>
       </item>
       <item>
        <widget class="QPushButton" name="ruleFileSelectB">
         <property name="text">
          <string>Select</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="wordlistGroupBox">
      <property name="title">
       <string>Wordlist:</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLineEdit" name="wordlistLE">
         <property name="placeholderText">
          <string>../dictionaries/*.txt</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="wordListSelectB">
         <property name="text">
          <string>Select</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="statusGroupBox">
      <property name="title">
       <string>Status</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QFormLayout" name="formLayout">
         <item row="1" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>&lt;b&gt;Time Started&lt;/b&gt;:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="startTimeL">
           <property name="text">
            <string>N/A</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Time Estimated:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="etaL">
           <property name="text">
            <string>N/A</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>&lt;b&gt;Speed:&lt;/b&gt;</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLabel" name="speedL">
           <property name="text">
            <string>N/A</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>&lt;b&gt;Status:&lt;/b&gt;</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="statusL">
           <property name="text">
            <string>N/A</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>&lt;b&gt;Password:&lt;/b&gt;</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLabel" name="passwordL">
           <property name="text">
            <string>N/A</string>
           </property>
           <property name="textInteractionFlags">
            <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QProgressBar" name="progressBar">
           <property name="value">
            <number>24</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="startStopB">
           <property name="toolTip">
            <string>StartStop</string>
           </property>
           <property name="text">
            <string>START</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pauseB">
           <property name="toolTip">
            <string>Pause</string>
           </property>
           <property name="text">
            <string>PAUSE</string>
           </property>
           <property name="icon">
            <iconset theme="QStyle::SP_BrowserReload">
             <normaloff>.</normaloff>.</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>522</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="actionAbout"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuAbout"/>
  </widget>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionOpenList">
   <property name="text">
    <string>Open list</string>
   </property>
  </action>
  <action name="action_Add">
   <property name="text">
    <string>&amp;Add</string>
   </property>
  </action>
  <action name="actionImport">
   <property name="text">
    <string>Import</string>
   </property>
  </action>
  <action name="actionUpdate">
   <property name="text">
    <string>Update</string>
   </property>
  </action>
  <action name="actionVHubsSend">
   <property name="text">
    <string>Send</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>?Help</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>66</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
